appliance_hardening_guide/_topics/db_ssl.md

```diff
@@ -74,30 +58,33 @@ To configure SSL on the database appliance:
 
     This changes the incoming communication protocol to use SSL and refuse any unencrypted PostgreSQL connections.
 
-9.  Start the `postgresql` and `evmserverd` services so that the changes take effect:
+7.  Start the `postgresql` and `evmserverd` services so that the changes take effect:
+
+        $ systemctl start postgresql.service
+        $ systemctl start evmserverd
 
-        [root@{{ site.data.product.title_short_l }}1 ~]# systemctl start postgresql.service
-        [root@{{ site.data.product.title_short_l }}1 ~]# systemctl start evmserverd
+#### Configure SSL on all connected appliances
 
 The database appliance now accepts only connections from connecting appliances that use SSL. The following procedure sets up connecting appliances to communicate to the database by using SSL. Use this procedure for each connecting appliance:
 
 1.  Log in as `root` to the connecting appliance.
 
-2.  Create the `.postgresql` directory in your `root` user home directory.
-
-        [root@{{ site.data.product.title_short_l }}2 ~]# mkdir /root/.postgresql
+2.  Copy the root certificate file from the database to the operating system's list of valid certificates:
 
-    The PostgreSQL client library, which {{ site.data.product.title_short }} also uses, looks to this directory for custom configuration files.
+        $ DATABASE=[database_appliance_fqdn]
+        $ scp root@${DATABASE}:/var/www/miq/vmdb/certs/root.crt /etc/pki/ca-trust/source/anchors/${DATABASE}-postgres
+        $ update-ca-trust extract
+        $ export PGSSLROOTCERT=/etc/pki/tls/certs/ca-bundle.crt
 
-3.  Copy the root certificate file from the database appliance to the `/root/.postgresql` directory on the connecting appliance:
+    Where `[database_appliance_fqdn]` is the fully qualified domain name of the database appliance.
 
-        [root@{{ site.data.product.title_short_l }}2 ~]# scp root@[database_appliance_fqdn]:/var/www/miq/vmdb/certs/root.crt /root/.postgresql/root.crt
+3.  Test the certificate got added to the certificate database:
 
-    Where `[database_appliance_fqdn]` is the fully qualified domain name of the database appliance.
+        $ openssl x509 -in /etc/pki/tls/certs/ca-bundle.crt -text -noout
 
 4.  Test the connection between the connecting appliance and the database appliance by using the `psql`command:
 
-        [root@localhost ~]# psql -h [database_appliance_fqdn] -d vmdb_production
+        $ PGSSLMODE=verify-full psql -h [database_appliance_fqdn] -d vmdb_production
```

Comment:
I wonder if it's simpler, instead of psql followed by a separate doc line to do \q, to just have a single line something like:

```sh
$ PGSSLMODE=verify-full psql -h [database_appliance_fqdn] -d vmdb_production -c "SELECT * FROM pg_stat_ssl"
```

or perhaps filtered by only the current pid if there is more than one connection:

```sh
$ PGSSLMODE=verify-full psql -h [database_appliance_fqdn] -d vmdb_production -c "SELECT * from pg_stat_ssl WHERE pid=pg_backend_pid()"
```